update.php

```diff
@@ -1145,3 +1145,33 @@ function update_1502()
 	DBA::e("UPDATE `pconfig` SET `cat` = 'calendar' WHERE `k` = 'first_day_of_week'");
 	return Update::SUCCESS;
 }
+
+function update_1505()
+{
+	$conditions = [
+		"((`cat`  = ?) AND ((`k` LIKE ?) OR (`k` = ?) OR (`k` LIKE ?) OR (`k` = ?))) OR " .
+		"((`cat` != ?) AND  (`k` LIKE ?)) OR " .
+		"((`cat`  = ?) AND  (`k` LIKE ?))",
+		"system",
+		"post_update_%",
+		"worker_last_cleaned",
+		"last%",
+		"worker_daemon_mode",
+		"system",
+		"last_%",
+		"database",
+		"update_%",
+	];
+
+	$postUpdateEntries = DBA::selectToArray('config', ['cat', 'k', 'v'], $conditions);
+
+	foreach ($postUpdateEntries as $postUpdateEntry) {
+		if ($postUpdateEntry['cat'] === 'system') {
+			DI::keyValue()->set($postUpdateEntry['k'], $postUpdateEntry['v']);
+		} else {
+			DI::keyValue()->set(sprintf('%s_%s', $postUpdateEntry['cat'], $postUpdateEntry['k']), $postUpdateEntry['v']);
+		}
+	}
+
+	return DBA::delete('config', $conditions) ? Update::SUCCESS : Update::FAILED;
```

Comment:
Why are you deleting the entries?

Reply:
They aren’t used anymore? Although the whole table wouldn’t be used anymore so the deletion is moot.

Reply:
Why do you want to keep them? We already store it at `key-value`. And in the worst case, we do have support requests with wrong `config` entries, because they're getting more and more wrong over time

Reply:
Why would the `config` table be used for after this PR?

Reply:
I'm a little bit confused now.

- This statement deletes all moved key-value pairs after the migration, so these entries aren't at two places
- This statement does not wipe/delete the whole `config` table

Reply:
Ah, my bad, I confused with the Neon PR which effectively obsoletes the config table.